sdk/textanalytics/Azure.AI.TextAnalytics/tests/Infrastructure/TextAnalyticsClientLiveTestBase.cs

```diff
@@ -43,5 +51,23 @@ public TextAnalyticsClientLiveTestBase(bool isAsync, TextAnalyticsClientOptions.
                 return InstrumentClient(new TextAnalyticsClient(endpoint, credential, InstrumentClientOptions(options)));
             }
         }
+
+        // This has been added to stop the custom tests to run forever while we
+        // get more reliable information on which scenarios cause timeouts.
+        // Issue https://github.com/Azure/azure-sdk-for-net/issues/25152
+        internal async Task PollUntilTime(AnalyzeActionsOperation operation)
```

Comment:
```suggestion
        internal async Task PollUntilTimeout(AnalyzeActionsOperation operation)
```

Should we allow an optional timeout to be passed to this, in case we need a one-off change to the default?    That said, to Caio's point earlier, if this is just temp infrastructure then feel free to ignore me.

Reply:
Ideally, we can delete this soon. But we never know... so adding a variable in case we want another value